landoapi/repos.py

```diff
@@ -281,6 +288,12 @@ def autoformat_enabled(self) -> bool:
             url="https://hg.mozilla.org/mozilla-build",
             access_group=SCM_LEVEL_3,
         ),
+        "beta": Repo(
+            tree="",
+            url="https://hg.mozilla.org/releases/mozilla-beta",
+            access_group=SCM_ALLOW_DIRECT_PUSH,
```

Comment:
Are we technically overloading this access group? I imagine this might be flagged by security team as something that should have an explicit access group.

Reply:
We could look at creating a separate access group to tighten restrictions on the number of people who can push vs the number of people who can land uplifts through this new workflow. For now this just reflects the current state of the world, we only want relman + sheriffs and a select few others (releng etc) doing uplift landings. I'll file a follow up bug for this.